src/deluge/gui/views/audio_clip_view.h

```diff
@@ -51,9 +51,8 @@ class AudioClipView final : public ClipView, public ClipMinder {
 	uint32_t getMaxLength();
 	uint32_t getMaxZoom();
 
-#if HAVE_OLED
+	// OLED ONLY
```

Comment:
I think comment like this is redundant when all functions/variables below already have "oled" in the name.

Reply:
Fixed.